pkgs/os-specific/linux/new-lg4ff/default.nix

```
@@ -30,6 +31,7 @@ stdenv.mkDerivation rec {
       KVERSION="${kernel.modDirVersion}"
       KDIR="${kernel.dev}/lib/modules/${kernel.modDirVersion}/build"
       KCFLAGS="-DCONFIG_LOGIWHEELS_FF -DCONFIG_LEDS_CLASS"
+      ${builtins.toString kernelModuleMakeFlags}
```

If this is a list, then I guess what you wanted was:
```suggestion
      ${builtins.concatStringsSep " " kernelModuleMakeFlags}
```